include/dpp/channel.h

```diff
@@ -158,6 +187,51 @@ struct DPP_EXPORT thread_member
 	 thread_member& fill_from_json(nlohmann::json* j);
 };
 
+/**
+ * @brief Specifies the emoji to use as the default way to react to a forum post. Exactly one of `emoji_id` and `emoji_name` must be set.
+ */
+struct DPP_EXPORT default_reaction_emoji {
+	/// The id of a guild's custom emoji
+	snowflake emoji_id;
+	/// The unicode character of the emoji
+	std::string emoji_name;
+};
```

Comment:
we could also add a variant here, as they're not both used at the same time iirc. ?

Reply:
> we could also add a variant here, as they're not both used at the same time iirc. ?

no, its both for a custom emoji and just the name for a built in emoji